### ISSUE TITLE
libtheora compilation error with newest cmake 3.9.1 and git 2.14.1

### ISSUE THREAD

**Issue by jasjuang** [Microsoft/vcpkg#1637]
I am not able to compile libtheora (vcpkg install libtheora ) with the newest cmake 3.9.1 and git 2.14.1. The error message is 

err.log
``````
ninja: error: manifest 'build.ninja' still dirty after 100 tries
``````

out.log
``````
[0/1] "C:\Program Files\CMake\bin\cmake.exe" -HC:\dev\vcpkg\buildtrees\libtheora\src\theora-fa5707d68c2a4338d58aa8b6afc95539ba89fecb -BC:\dev\vcpkg\buildtrees\libtheora\x64-windows-rel
-- Configuring done
-- Generating done
-- Build files have been written to: C:/dev/vcpkg/buildtrees/libtheora/x64-windows-rel

repeat for 100 times
``````

I was able to compile this with my old setup, cmake 3.9.0 and git 2.11. I am still trying to figure out why ninja is causing trouble. Can someone give me some help?

EDIT: If I comment out the PREFER_NINJA in the portfile it compiles, but I am sure this is probably not the desired way to fix it

**Comment by HaochengLIU** [comment]
There is a CMake [issue](https://gitlab.kitware.com/cmake/cmake/issues/17191) now. Add `chcp 65001` to local consol(using UTF-8 encoding) should be a work around solution.

**Comment by jasjuang** [comment]
Solved by commit 67b9475
